example/data-migration-ecs/terraform.tf.example

```diff
@@ -0,0 +1,8 @@
+terraform {
+  backend "s3" {
+    region         = "us-east-1"
+    bucket         = "PREFIX-state"
+    key            = "PREFIX/data-migration-ecs/terraform.tfstate"
+    dynamodb_table = "PREFIX-tf-locks"
```

Comment:
This is a nit but should this be `PREFIX-locks`?

Reply:
Yeah, totally agreed that it would probably be clearer as `PREFIX-locks`. I left it as `PREFIX-tf-locks` because it looks like that's the format we're using in our other deploys. `cumulus-tf/terraform.tf.example` for example:

`dynamodb_table = "PREFIX-tf-locks"`

I think if you were to make your prefix `jt-tf` you'd end up with `jt-tf-tf-locks`. I wouldn't think that would matter. It would just look a little wonky.